src/Convert/Convert.php

```diff
@@ -100,11 +100,17 @@ public function toDMS($format = ConvertInterface::DEFAULT_DMS_FORMAT)
     /**
      * {@inheritDoc}
      */
-    public function toDecimalMinutes($format = ConvertInterface::DEFAULT_DM_FORMAT)
+    public function toDecimalMinutes($format = ConvertInterface::DEFAULT_DM_FORMAT, $precision = false)
```

Comment:
Lets not mix types. We can use `$precision = 0` as default

Reply:
Can I use `null` instead of `false` ?

If `precision` equal `null`, it's the default behaviour. With `precision` as an int, it's use to format the decimal part. Users could use `0` for no precision, or int >0 for the precision they want

Reply:
What precision do you get if you use null?

Yes, null is better than false.

Reply:
Default value is 5 : https://github.com/thephpleague/geotools/blob/master/src/Convert/ConvertInterface.php#L26
Maybe it's better to override this value when I want only 3 decimal for the DDM format.

Reply:
This PR could be split in two part : custom precision for conversion, and this new format.